src/gazebo_mavlink_interface.cpp

```diff
@@ -208,21 +208,6 @@ void GazeboMavlinkInterface::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf
           {
             std::string joint_name = channel->Get<std::string>("joint_name");
             joints_[index] = model_->GetJoint(joint_name);
-            if (joints_[index] == nullptr)
-            {
-              gzwarn << "joint [" << joint_name << "] not found for channel["
```

Comment:
I would keep this as a gzdbg.

Reply:
Why? Is it an error? Or just for development?

Reply:
Development